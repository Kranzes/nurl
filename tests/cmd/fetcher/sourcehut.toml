args = ["https://git.sr.ht/~sircmpwn/core.sr.ht", "0.60.0"]
