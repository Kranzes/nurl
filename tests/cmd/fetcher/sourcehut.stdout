fetchFromSourcehut {
  owner = "~sircmpwn";
  repo = "core.sr.ht";
  rev = "0.60.0";
  hash = "sha256-d2lfBMf5dJeqvTT6NWPNDSuYC3Hgof8PanYwSHqOCEk=";
}